Src/BootCamp.Chapter/OutputTime.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+
+    namespace BootCamp.Chapter
+    {
+        public class OutputTime
```

Comment:
Output is a horrible name. Please rename it so that it represent business logic.